test/no-array-for-each.js

```diff
@@ -5,7 +5,9 @@ test.snapshot({
 	valid: [
 		'new foo.forEach(element => bar())',
 		'forEach(element => bar())',
-		'foo.notForEach(element => bar())'
+		'foo.notForEach(element => bar())',
+		// #1087
+		'React.Children.forEach(children, (child) => {});'
```

Comment:
This could also be `Children.forEach(children, (child) => {});` with named import.